Lib/test/test_msvcrt.py

```diff
@@ -61,34 +61,47 @@ def test_get_osfhandle(self):
 
 
 class TestConsoleIO(unittest.TestCase):
+    def run_in_separated_process(self, code):
+        # Run test in a seprated process to avoid stdin conflicts.
+        # See: gh-110147
+        cmd = [sys.executable, '-c', code]
+        subprocess.run(cmd, check=True, capture_output=True,
+                       creationflags=subprocess.CREATE_NEW_CONSOLE)
+
+
+    @requires_resource('gui')
```

Comment:
I suggest moving the decorator to run_in_separated_process() and explain why you require GUI resource. For example, say that CREATE_NEW_CONSOLE creates a "popup" window.

Reply:
Done.